src/v3/plugins/github/parseReferences.js

```diff
@@ -24,17 +17,33 @@ export function parseReferences(body: string): string[] {
   ];
 }
 
-function findNumericReferences(body: string): string[] {
-  return findAllMatches(/(?:\W|^)(#\d+)(?:\W|$)/g, body).map((x) => x[1]);
+function findNumericReferences(body: string): ParsedReference[] {
+  return findAllMatches(/(?:\W|^)(#\d+)(?:\W|$)/g, body).map((x) => ({
+    refType: "basic",
+    ref: x[1],
+  }));
 }
 
-function findUsernameReferences(body: string): string[] {
-  return findAllMatches(/(?:\W|^)(@[a-zA-Z0-9-]+)(?:\W|$)/g, body).map(
-    (x) => x[1]
-  );
+function findUsernameReferences(body: string): ParsedReference[] {
+  const pairedWithRefs = findAllMatches(
+    /(?:\W|^)(?:P|p)aired (?:w|W)ith:? (@[a-zA-Z0-9-]+)(?:\W|$)/g,
```

Comment:
Appreciate the leniency in the regexp. Could you also permit the space
between “paired” and “with” to be a single hyphen, like `Paired-with`?
This makes the format compatible with `git-interpret-trailers`, which
uses keys like `Signed-off-by`.

Reply:
Done.